app/controllers/index.rb

```diff
@@ -0,0 +1,155 @@
+get '/' do
+  session[:id]
+  erb :index
+end
+
+get '/browse' do
+  @all_dogs = Dog.all.sort
+  erb :browse
+end
+
+get '/search' do
+  erb :search
+end
+
+get '/search/results' do
+  @breed_list = Dog.where(breed: params[:breed])
+  p @breed_list.empty?
+  erb :search_results
+end
+
+get '/login/admin' do
+
+  erb :loginadmin
+end
+
+post '/login/admin' do
+  @admin = Admin.find_by(username: params[:username])
+  if @admin && @admin.password == params[:password]
+    session[:admin_id] = @admin.id
+  else
+    redirect '/login/admin'
+  end
+  redirect '/options'
+end
+
+get '/login' do
+  erb :login
+end
+
+post '/login' do
+  @volunteer = Volunteer.find_by(username: params[:username])
+  #if the username matches the db, then, redirect them to the main page
+  if @volunteer && @volunteer.password == params[:password]
+    session[:volunteer_id] = @volunteer.id #it stores the volunteer_id, which indicates they are logged in so they don't have to keep logging in and you don't have to keep looking their volunteer info when they go to a different page
```

Comment:
using `session[:admin_id]`  and `session[:volunteer_id]` would allow someone to be signed in as two difference users at the same time. Is this intentional?